packages/client3/utilities/contract.util.ts

```diff
@@ -0,0 +1,13 @@
+import { ContractReceipt } from "ethers";
+import { Result } from "ethers/lib/utils";
+
+export function getLastEventArgs(result: ContractReceipt): Result {
+  const events = result.events;
+  if (events) {
+    const lastEvent = events[events.length - 1];
+    if (lastEvent) {
+      return lastEvent.args as Result;
```

Comment:
To clarify isn't the type automatically inferred by Typescript? If it is I would suggest removing explicit typing.

Reply:
I think I was getting a typescript error, but I can double check.